src/jloptions.c

```diff
@@ -652,6 +651,8 @@ JL_DLLEXPORT void jl_parse_opts(int *argcp, char ***argvp)
             if (!strcmp(optarg,"ieee"))
                 jl_options.fast_math = JL_OPTIONS_FAST_MATH_OFF;
             else if (!strcmp(optarg,"fast"))
+                jl_options.fast_math = JL_OPTIONS_FAST_MATH_OFF;
```

Comment:
```suggestion
                ;
```
This shouldn't set it to `OFF`, since that makes it equivalent to `--math-mode=ieee`. It should just do nothing. We should probably also give a deprecation warning here.

Reply:
I thought `--math-mode=ieee` was the default, so to disable "fast", how else should this be?

Reply:
I see it now, I think, there's JL_OPTIONS_FAST_MATH_DEFAULT for the undocumented "user" option...